src/panel.js

```diff
@@ -0,0 +1,19 @@
+/* global browser */
+
+document.addEventListener("DOMContentLoaded", async () => {
+  if (document.querySelector("#panel1") !== null) {
+    await browser.storage.local.set({PANEL_SHOWN: true});
+    await browser.browserAction.setBadgeText({text: ""});
+  }
+
+  const tabsQueryResult = await browser.tabs.query({currentWindow: true, active: true});
+  const currentActiveTab = tabsQueryResult[0];
+  const currentActiveURL = new URL(currentActiveTab.url);
+  const currentActiveHostname = currentActiveURL.hostname;
```

Comment:
`hostname` will be empty for urls starting with `about:` and hence the header text in the popup wont start with an url.